<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>System In View</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>740</width>
      <height>350</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>740</width>
      <height>350</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>740</width>
      <height>350</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="basictab">
     <attribute name="title">
      <string>BasicInfo</string>
     </attribute>
     <widget class="QLabel" name="ker_ver_lb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Utopia</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Kernel Version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="ker_ver_contex">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>CM Roman Asian</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="hostname_lb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Utopia</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Host Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hostname_contex">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>CM Roman Asian</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="process_lb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Utopia</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Processor:</string>
      </property>
     </widget>
     <widget class="QLabel" name="process_contex">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>CM Roman Asian</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="basic_pic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>700</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>bg.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>720</width>
        <height>301</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="memtab">
     <attribute name="title">
      <string>Mem&amp;&amp;Disk</string>
     </attribute>
     <widget class="QTableWidget" name="memwidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>740</width>
        <height>100</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>740</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>740</width>
        <height>100</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>37</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>Capacity(KB)</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Capacity(MB)</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </row>
      <column>
       <property name="text">
        <string>MemTotal</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MemFree</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Buffers</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cached</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SwapTotal</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SwapFree</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
      <item row="0" column="2">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
      <item row="0" column="3">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
      <item row="0" column="4">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
      <item row="0" column="5">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </item>
      <item row="1" column="1">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
      <item row="1" column="2">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
      <item row="1" column="3">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
      <item row="1" column="4">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
      <item row="1" column="5">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="disk_total_num">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>CM Roman Asian</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="disk_free_num">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>CM Roman Asian</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="diskwidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>741</width>
        <height>181</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Usage</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DeviceName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mount Point</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Filesystem</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>size(GB)</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="loadtab">
     <attribute name="title">
      <string>LoadAvg</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>49</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>CM Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1  Minute</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>49</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>CM Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5  Minutes</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>CM Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>15  Minutes</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>531</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdNumber_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_1">
         <property name="font">
          <font>
           <family>CM Roman</family>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="processtab">
     <attribute name="title">
      <string>Process</string>
     </attribute>
     <widget class="QTableWidget" name="processtable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>300</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nice</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mem</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="total_pro">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>Total Process:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="run_pro">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>80</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Running:</string>
      </property>
     </widget>
     <widget class="QLabel" name="sleep_pro">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>150</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>Sleeping:</string>
      </property>
     </widget>
     <widget class="QLabel" name="zom_pro">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>220</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>Zombie:</string>
      </property>
     </widget>
     <widget class="QLabel" name="total_num">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>75</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="run_num">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>110</y>
        <width>75</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="sleep_num">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>180</y>
        <width>75</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="zom_num">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>250</y>
        <width>75</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_finish">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>280</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Kill process</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="nettab">
     <attribute name="title">
      <string>NetWorks</string>
     </attribute>
     <widget class="QTableWidget" name="nettable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>231</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pingButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>260</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Ping</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="devtab">
     <attribute name="title">
      <string>DevInfo</string>
     </attribute>
     <widget class="QTableWidget" name="devtable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>740</width>
        <height>320</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuCheckConfig">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>CheckConfig</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="separator"/>
    <addaction name="actionReboot"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string> About</string>
    </property>
    <addaction name="actionGroup"/>
   </widget>
   <addaction name="menuCheckConfig"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionReboot">
   <property name="text">
    <string>Reboot</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="text">
    <string>Group</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>memwidget</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGroup</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
